asapdiscovery/simulation/scripts/MproP0009.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Jan 28 18:10:15 2023
+
+@author: kendalllemons
+"""
+
+import os
+import openmm
+import argparse
+
+
+
+# This script was generated by OpenMM-Setup on 2023-01-17.
+
+from openmm import *
+from openmm.app import *
+from openmm.unit import *
+from openmm.app import PDBFile
+
+
+## Parameters
+def get_args():
+    parser = ArgumentParser()
+
+    ## Input arguments
+    parser.add_argument(
+        "-i", 
+        "--input_pdb_path", 
+        type = str,
+        default = "/data/chodera/asap-datasets/full_frag_prepped_mpro_12_2022/Mpro-P0009_0A_MAT-POS-f2460aef-1/prepped_receptor_0.pdb",
+        help = "Path to PDB file to simulate"
+    )
+    parser.add_argument(
+        "-o", 
+        "--output_dir",
+        type = str,
+        default = "/data/chodera/lemonsk/default_out",
+        help="Output simulation directory."
+    )
+    args = parser.parse_args()
+    return args
+
```

Comment:
You need to add a line:
```
args = get_args()
```
in order for `args.input_pdb_path` to actually work

Reply:
you could also just put everything in the `get_args` function in the main script. which is a bit simpler.